utils/logger.go

```diff
@@ -15,6 +17,22 @@ const (
 	LOG_CALLER_SKIP      = 3 // stack frame depth
 )
 
+type contextKey string
+
+const loggerKey = contextKey("logger")
```

Comment:
not that this isn't a bad thing but why introduce it with this PR? the place you use it (api/crio.go) still has access to s.logger. Does ctx need a logger for some other reason?

Reply:
I have it being picked up in api/crio/crio.go through ctx, I didn't want to add it as another argument